src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -8338,8 +8322,7 @@ class Program
 {
     static void M(R r0)
     {
-        scoped R r1;
-        ref readonly scoped R r2 = ref r1;
```

Comment:
>ref readonly scoped [](http://example.com/codeflow?start=8&length=19)

We should include a `ref readonly` example. Perhaps change `r3` to `scoped ref readonly R r3;`